zebra-consensus/src/chain/tests.rs

```diff
@@ -350,6 +250,12 @@ async fn verify_fail_add_block_checkpoint() -> Result<(), Report> {
     Ok(())
 }
 
+/*
+// This test is disabled because it doesn't test the right thing:
+// the BlockVerifier and CheckpointVerifier make different requests
+// and produce different transcripts.
+
+
 #[tokio::test]
 // Temporarily ignore this test, until the state can handle out-of-order blocks
```

Comment:
Do we want to remove this?  It's still a blocker (among the other blockers, xref https://github.com/ZcashFoundation/zebra/pull/1043#discussion_r487236237 ) since the state doesn't handle out of order `CommitBlock` requests.

Reply:
I think it's fine, just slightly confusing to have two reasons to disable a test in different comments